packages/airnode-validator/src/config/config.ts

```diff
@@ -319,7 +344,9 @@ export const configSchema = z
   .strict()
   .superRefine(validateSecuritySchemesReferences)
   .superRefine(validateTemplateSchemes)
-  .superRefine(validateTriggersReferences);
+  .superRefine(validateTriggersReferences)
+  // TODO: Add check that the security schemes referenced in security are present
```

Comment:
@Siegrift Do we track this somewhere?

Reply:
Not really. That one is checking that if the security scheme is specified you have API credentials specified as well. I'm talking about a check that if you have something defined in `security` it needs to be available in `components.securitySchemes`. This check will just ignore if the security scheme defined in `security` doesn't exist.

Reply:
I see. This is checked in https://github.com/api3dao/airnode/blob/reject-relay-metadata-gateway/packages/airnode-validator/src/ois/ois.ts#L178

Reply:
Oh, ok, didn't notice that. In that case, I will have to change the code a bit. I thought that if it's caught early on I don't need to check for the field existence. But apparently the check is there and I still need to :smile: